fe/fe-core/src/main/java/org/apache/doris/nereids/jobs/batch/AnalyzeRulesJob.java

```diff
@@ -55,7 +56,8 @@ public AnalyzeRulesJob(CascadesContext cascadesContext, Optional<Scope> scope) {
                         new ProjectToGlobalAggregate()
                 )),
                 topDownBatch(ImmutableList.of(
-                        new ResolveAggregateFunctions()
+                        new ResolveAggregateFunctions(),
+                        new ResolveSortProject()
```

Comment:
Rename to BindXxx?

Reply:
it is not a bind action. actually, this rule add the projections need by LogicalSort to it's child Project's projections and add another Project on the top of itself to ensure the result is correct

Reply:
FillupMissingSlots